client/components/date-picker/docs/example.jsx

```diff
@@ -23,7 +23,7 @@ class DatePickerExample extends Component {
 
 			{
 				title: 'Social Media - Facebook',
-				date: new Date(),
+				date: new Date( +new Date() + 60 * 60 * 24 * 1000 ),
```

Comment:
I'm not opposed to it, but I just learned about using the unary `+` operator in this way. Could it be more clear if we used `Date.now()`, which I think does the same thing as `+new Date()`? Or maybe is there some advantage to this style that I'm not aware of?

Also, I know this is just the example, so it's not super-important, but I was curious.